File: Solutions/ErenAbhayMichael/Clements 2025/Venue.java
import java.util.*;
import java.io.*;

public class Venue{
    public static void main(String[] args) throws IOException{
        Scanner sc = new Scanner(new File("venue.dat"));
        int m = sc.nextInt();
        int h = sc.nextInt();
        sc.nextLine();
        for (int i = 0; i < m; i++) {
            String name = sc.next();
            double cost = sc.nextDouble();
            int l = sc.nextInt();
            int w = sc.nextInt();
            sc.nextLine();
            System.out.printf(name + " " + "%.6f%n", (cost/h/l/w));
        }
    }
}

File: venue.py
with open("Input/venue.dat") as f:
    m, h = map(int, next(f).split())
    for line in f:
        name, p, l, w = line.split()
        p = float(p)
        l, w = map(int, [l, w])

        print(f"{name} {(p/(l * w * h)):.6f}")